cli/cli.go

```diff
@@ -22,13 +22,15 @@ type Format struct {
 	Version               bool               `name:"version" short:"V" help:"Print version."`
 	Init                  bool               `name:"init" short:"i" help:"Create a new treefmt.toml."`
 
+	OnUnmatched log.Level `name:"on-unmatched" short:"u" default:"warn" help:"Log paths that did not match any formatters at the specified log level. Possible values are debug,info,warn,error,fatal."`
```

Comment:
```suggestion
	OnUnmatched log.Level `name:"on-unmatched" short:"u" default:"debug" help:"Log paths that did not match any formatters at the specified log level. Possible values are debug,info,warn,error,fatal."`
```
The default was debug right?

Reply:
@zimbatm yeah I changed it to `WARN` because I think it's useful on first un-cached application to see files you missed.

Reply:
Once the cache is hot, you won't get the warnings, since the file is known and won't have changed, therefore it doesn't get processed.

Reply:
nice, ok that makes sense